### ISSUE TITLE
I have a question

### ISSUE THREAD

**Issue by DumberSoft** [BobHasNoSoul/jellyfin-mods#34]
In the login how can I add another button and how to remove the reset password.

**Comment by BobHasNoSoul** [comment]
What button are you speaking about adding? Like a new custom button or
something else? (I ask because some are different to others)

And to hide the reset password button

In custom CSS inside admin panel


button.raised.cancel.block.btnForgotPassword.emby-button {display:none
!important}


Save it and then reload the cache of the page

**Comment by DumberSoft** [comment]
Hello, thank you for responding, well what I'm going for is a custom button, for example I want to eliminate the one that says change server and forgotten password and there put another custom button that, for example, redirects to another page such as a store or personalized thing. 
![IMG_20240814_071220](https://github.com/user-attachments/assets/85f66207-d7f3-4420-b295-a1e28ccc82da)

**Comment by BobHasNoSoul** [comment]
Ahh I see so it's not a button to an item in that case yeah that's easy
give me a few days (work and family stop me from doing a lot of mods) but
will write something up

**Comment by BobHasNoSoul** [comment]
please see https://github.com/BobHasNoSoul/jellyfin-mods/blob/main/10.9.x.md#adding-a-new-custom-link-button-under-login-page-109x 

this should do what you want

note you will need the change the example.com and the text on the button to your own link / text to give you the desired effect (note i have no idea what the use of this would be but added it there anyway)

**Comment by DumberSoft** [comment]
Oh thank you very much, but to change the format of changing the server that appears on mobile phones?

**Comment by BobHasNoSoul** [comment]
sorry that didnt really make sense to me can you try explaining in greater detail?

do you mean to hide the change server button?  the part that has thrown me is the mobile phone part

**Comment by DumberSoft** [comment]
Yes, I would like to know for mobiles because on mobiles it is not the same as on the web since on the mobile the change of server appears in login But on the web it does not appear.

**Comment by BobHasNoSoul** [comment]
ohh so you just need to hide it, put this in your custom css block in admin page

#loginPage button.raised.cancel.block.btnSelectServer.emby-button {display:none
!important}

this removes it from the login page (using #loginPage selector so it doenst break the other places for changing the server like network failure.

**Comment by DumberSoft** [comment]
I have already set the delete button and the option still appears.
![Uploading IMG_20240817_124006.jpg…]()

**Comment by DumberSoft** [comment]
![Uploading IMG_20240817_124006.jpg…]()

**Comment by DumberSoft** [comment]
![IMG_20240817_124006](https://github.com/user-attachments/assets/6e469cb7-e082-44d2-b754-6c0301187eca)

**Comment by BobHasNoSoul** [comment]
it shouldnt appear that is odd, can you clear the cache of the device and try again or use private browsing mode to load the page without using the cached files?

**Comment by BobHasNoSoul** [comment]
![Screenshot 2024-08-17 193527](https://github.com/user-attachments/assets/15bcbae9-35a4-40c4-9093-abcc309bf6ee)

**Comment by DumberSoft** [comment]
I have a question, how do I change the title and the others for Windows since I see that it is only for Linux.

**Comment by BobHasNoSoul** [comment]
You can use windows subsystem Linux and follow the same instructions but
substitute your c drive with /mnt/c

**Comment by DumberSoft** [comment]
Sorry but I don't know how to do it, do you have some examples or explanations please, to change the titles and names please.
